app/models/share.rb

```diff
@@ -1,12 +1,14 @@
 class Share < ActiveRecord::Base
   belongs_to :groups
 
-  attr_accessor :activation_token
+  attr_accessor :activation_token, :password_reset_token
 
   before_create :create_activation_digest
 
   validates :name, presence: true, length: { minimum: 3 }
-  validates :email, presence: true, format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i }
+  validates :email, presence: true,
+    format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i },
+    uniqueness: true
```

Comment:
hier wird's kleine Konflikte mit dem Master geben, weil ich diese Validation neulich in ähnlicher Form in den Master gemerged habe. Wirst du aber leicht checken...